lib/intacct/actions/create.rb

```diff
@@ -26,10 +26,10 @@ def request(_options)
       end
 
       def response_body
-        raw = @response.at("//result/data/#{klass.api_name}")
+        raw = @response.at("//result/data/#{klass.api_name.downcase}")
```

Comment:
I'm not sure this is a good idea as I know Intacct endpoints are not consistent in their case.

Reply:
We've tested it with Bill, Expense, Project, Timesheet, and Task and they have all worked. The only other endpoint that we create is Observed Percent Completed but that doesn't use the output of the create call at all so it shouldn't matter

Reply:
Sounds good! Thanks.